.githooks/pre-commit/lint

```diff
@@ -21,8 +21,8 @@ trap cleanup EXIT SIGINT SIGHUP
 git diff --no-prefix > "$PATCH_FILE"
 git checkout -- .
 
-git_cached_files=$(git diff --cached --name-only --diff-filter=ACMR | xargs echo)
+git_cached_files=$(git diff --cached --name-only --diff-filter=ACMR | grep \.js$ -e \.bemhtml$ -e \.bemtree$ | xargs echo)
```

Comment:
@zxqfox looks like the `-e` switch has been missed before `\.js$` here. Current snippet leads to

> grep: \.js$: No such file or directory

in case of no JS files was modified.